src/main/java/kappzzang/jeongsan/dto/response/TeamResponse.java

```diff
@@ -0,0 +1,29 @@
+package kappzzang.jeongsan.dto.response;
+
+import kappzzang.jeongsan.domain.Team;
+import kappzzang.jeongsan.dto.MemberPreview;
+
+import java.util.List;
+
+public record TeamResponse(
+        Long teamId,
+        String name,
+        Boolean isCompleted,
+        String subject,
+        List<MemberPreview> memberPreviews
+) {
+
+    public static TeamResponse from(Team team) {
+        return new TeamResponse(
+                team.getId(),
+                team.getName(),
+                team.getIsClosed(),
+                team.getSubject(),
+                team.getTeamMemberList()
+                        .stream()
+                        .limit(3)
```

Comment:
P3: 가져올 TeamMember Preview 개수의 경우 상수로 따로 빼서 관리해도 좋을 거 같습니다!
```suggestion
                        .limit(MAX_PREVIEW_MEMBERS)           
```

Reply:
좋은 것 같습니다! 반영하겠습니다